utils/release_helper.py

```diff
@@ -0,0 +1,786 @@
+    branches = get_repo_branches(repo)
+    return filter_repo_branch_by_name(branches, "stable")
+
+
+def get_old_stabilization_branch(repo) -> object:
+    branches = get_repo_branches(repo)
+    latest_version = get_latest_version(repo)
+    branch_name = f'stabilization-v{latest_version}'
+    return filter_repo_branch_by_name(branches, branch_name)
+
+
+def remove_old_stabilization_branch(repo, branch) -> None:
+    if get_confirmation(f'Are you sure about removing the "{branch.name}" branch?'):
+        # https://github.com/PyGithub/PyGithub/issues/1570
+        try:
+            branch_ref = repo.get_git_ref(f'heads/{branch.name}')
+            branch_ref.delete()
+        except Exception as e:
+            print(f'Error: {e}')
+            exit(1)
+    else:
+        print(f'Aborted! {branch.name} branch not removed.')
+
+
+# Repository Contributors
+def get_contributors_commit_diff(commit) -> str:
+    commit_diff = subprocess.run(
+        ['git', 'show', '--word-diff', commit, 'Contributors.md'],
+        capture_output=True, text=True)
+    return commit_diff.stdout
+
+
+def get_contributors_last_commit() -> str:
+    last_commit = subprocess.run(
+        ['git', 'log', '--pretty=format:%h', '-1', '--', 'Contributors.md'],
+        capture_output=True, text=True)
+    return last_commit.stdout
+
+
+def get_contributors_last_update() -> str:
+    last_commit = get_contributors_last_commit()
+    last_commit_diff = get_contributors_commit_diff(last_commit)
+
+    for line in last_commit_diff.split('\n'):
+        # The "generate_contributors.py" creates a commend in the beginning of the file.
+        # This comment informs when the file was updated.
+        if 'Last Modified:' in line:
+            elements = line.split('{+')
+            datetime = elements[1].replace('+}', '')
+            return datetime
+
+
+def get_new_contributors(commit) -> str:
+    last_commit_diff = get_contributors_commit_diff(commit)
+    new_contributors = []
+
+    for line in last_commit_diff.split('\n'):
+        if line.startswith('{+'):
+            for chars in ['{+', '+}']:
+                line = line.replace(chars, '')
+            new_contributors.append(line)
+    return '\n'.join(new_contributors)
+
+
+def is_contributors_list_updated(date_string) -> bool:
+    date = datetime.strptime(date_string, '%Y-%m-%d %H:%M')
+    # As a rule of thumbs, this function consider contributors list
+    # updated if not older than 2 weeks.
+    two_weeks_back = datetime.now() - timedelta(days=15)
+    if date < two_weeks_back:
+        return False
+    return True
+
+
+def update_contributors():
+    last_update = get_contributors_last_update()
+    if is_contributors_list_updated(last_update):
+        print(f'It is all fine, the contributors list was updated in {last_update}')
+    else:
+        print(f'Contributors list last update was in {last_update}. I can update it for you.')
+        root_dir = os.path.dirname(os.path.dirname(__file__))
```

Comment:
I am afraid this is a bit fragile... would it be possible to have this defined as a constant? Maybe source it from the file where the Github token is stored?

Reply:
True. I can make it more robust. But not based on the Github token file because it could also be fragile. I will take a look on this. Thanks.

Reply:
I believe it is much more robust now as all functions work as expected regardless where the script is called from.